pyxem/utils/io_utils.py

```diff
@@ -27,6 +27,8 @@
 from hyperspy.io import load_with_reader
 
 import numpy as np
+import dask.array as da
+from math import floor
 
 from pyxem.signals.diffraction1d import Diffraction1D
```

Comment:
Need to remove imports from io_plugins, i.e. delete the following:

from pyxem.io_plugins import io_plugins, default_write_ext
from pyxem.io_plugins import mib as mib_reader